pytest.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
 filterwarnings=
     error
+    # This is internal to the docker library.
+    ignore:the imp module is deprecated:DeprecationWarning
```

Comment:
It's best to use the fourth parameter (add yet another semicolon) to specify where exactly this exception is expected/allowed to originate from.